### ISSUE TITLE
GPUs are offline while CPUs are on same pc.

### ISSUE THREAD

**Issue by notevil1337** [UnamSanctam/SilentCryptoMiner#1893]
why it happens? i got 2 miner and their GPUs are offline and not mining
![image](https://github.com/UnamSanctam/SilentCryptoMiner/assets/141959915/5797177e-c8de-454a-888c-c879f24f0dc7)
![image](https://github.com/UnamSanctam/SilentCryptoMiner/assets/141959915/06c53a11-e49c-4dac-a3ff-379f4d5d75f2)

**Comment by UnamSanctam** [comment]
Usually it means that something is wrong with their GPU or their GPU drivers. It's not that uncommon. What GPUs do they have?

**Comment by notevil1337** [comment]
nope, they were both mining yesterday. they got nice GPUs tho
![image](https://github.com/UnamSanctam/SilentCryptoMiner/assets/141959915/b6772c68-7ab2-417d-ab7a-637772705dc1)
![image](https://github.com/UnamSanctam/SilentCryptoMiner/assets/141959915/c7e9d264-733d-45ec-b870-8ef775faf293)

**Comment by notevil1337** [comment]
second one is not nice but first one is xd

**Comment by UnamSanctam** [comment]
Yes then something with the GPU caused the miner to close (and cannot open again since I assume you have the watchdog enabled). It has happened a few times with my test GPUs, either when I have something really intensive running or just randomly. For my test GPUs they usually fixed themselves after a computer restart or if I reinstalled the GPU driver.

Do you have the watchdog enabled/installed? Since if you do then something like what I said above is the case, because if that wasn't the case the watchdog would just start the miners again.

**Comment by notevil1337** [comment]
yep i have watchdog installed? its okay, if they will get fixed after restart it will be good

**Comment by UnamSanctam** [comment]
Yes that should be the case, most likely the watchdog is repeatedly trying to start the GPU miner at the moment, so them being offline points to something with the GPU. And that usually gets fixed with a restart.
